src/pages/Bookmarked.tsx

```diff
@@ -21,6 +21,11 @@ import { Plugins } from '@capacitor/core';
 import {folder, notifications, notificationsOutline, notificationsCircleOutline} from 'ionicons/icons';
```

Comment:
This file is old. If you resolve merge conflicts, make sure to discard this branches version and accept all of the develop branches.